From 35e9c94fb265213848d4cd2b0d062dec62cdfb0a Mon Sep 17 00:00:00 2001
From: Arthur Demchenkov <spinal.by@gmail.com>
Date: Fri, 14 Dec 2018 16:36:04 +0300
Subject: [PATCH] ARM: dts: n900: remove rx51-battery

N900 has bq27200 chip, which provides much better
functionality when exposing battery properties.

No need to confuse userspace with two battery devices
exposed by the kernel at the same time.
---
 arch/arm/boot/dts/omap3-n900.dts | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/boot/dts/omap3-n900.dts b/arch/arm/boot/dts/omap3-n900.dts
index 826920e6b8787..de7a31ae72207 100644
--- a/arch/arm/boot/dts/omap3-n900.dts
+++ b/arch/arm/boot/dts/omap3-n900.dts
@@ -137,12 +137,6 @@
 		speaker-amplifier-gpios = <&twl_gpio 7 GPIO_ACTIVE_HIGH>;
 	};
 
-	battery: n900-battery {
-		compatible = "nokia,n900-battery";
-		io-channels = <&twl_madc 0>, <&twl_madc 4>, <&twl_madc 12>;
-		io-channel-names = "temp", "bsi", "vbat";
-	};
-
 	pwm9: dmtimer-pwm {
 		compatible = "ti,omap-dmtimer-pwm";
 		#pwm-cells = <3>;
